pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/sql.tsx

```diff
@@ -40,6 +40,26 @@ export default function (props: GraphDashboardProps) {
       </Axis>
     </LineGraph>,
 
+    <LineGraph
+      title="Open SQL Transactions"
+      sources={nodeSources}
+      tooltip={`The total number of SQL transactions currently open ${tooltipSelection}.`}
+    >
+      <Axis label="transactions">
+        <Metric name="cr.node.sql.txns.open" title="Open Transactions" />
```

Comment:
drive-by comment -- i'm curious if this includes open implicit transactions?

more fundamentally, i'm interested in how would i answer the question: "how many of my SQL connections are currently doing something?" which metric would i use for that? (no prob if the answer is none of these, i'm just trying to understand these metrics, not suggesting any changes)

Reply:
It does include implicit txns

Reply:
I think "doing something" would be satisfied by open txns, which cost resources even if the connection is idle. Maybe active queries would be slightly better, since those are doing "more work".